TS01-Time-Series_Metadata.Rmd

```diff
@@ -65,37 +60,20 @@ refresher on setting the working directory, please view the <a href="XXX" target
 
 </div>
 
-#Understand Your Data 
```

Comment:
What was cut? i'm happy to look at it. I found the content in the lesson to be too long generally. When we teach in person - the lessons online are a backup. i barely use them but do when there are images. it's more to prep me to teach. Online however people want to get to the point - how to do something and the basic concepts. I noticed in some instances of these lessons, the explanations were written out and could be considerably consolidated. Then you link to the technical explanation. 

If the text is too long, people won't get through the lesson. But again i'm saying this without knowing what was cut.